app/components/point-input.jsx

```diff
@@ -0,0 +1,185 @@
+'use strict'
+import React, { useEffect, useRef, useState } from 'react';
+import { createUseStyles } from 'react-jss';
+import cx from 'classnames';
+import autosize from 'autosize';
+
+function PointInput(props) {
+    const { style={}, className="", maxWordCount = 30, maxCharCount = 100, defaultValue = {description: "", subject: ""}, onDone = (valid=false, values = { subject: "", description: "" }) => ({valid, ...values}), ...otherProps} = props
+    const classes = useStyles()
```

Comment:
if it's not passed, onDone won't do anything so just say `onDone=()=>{}` here

Reply:
Great, I removed the default onDone completely based on this comment. Is this change now the functionality we want?